tests/end_to_end/test_brute_force/test_brute_force.py

```diff
@@ -0,0 +1,56 @@
+import os
+import pytest
+from tempfile import gettempdir
+
+from wazuh_testing.tools import configuration as config
+from wazuh_testing import end_to_end as e2e
+from wazuh_testing import event_monitor as evm
+
+# Test cases data
+alerts_json = os.path.join(gettempdir(), 'alerts.json')
+test_data_path = os.path.join(os.path.dirname(os.path.realpath(__file__)), 'data')
+test_cases_file_path = os.path.join(test_data_path, 'test_cases', 'cases_brute_force.yaml')
+
+# Playbooks
+configuration_playbooks = ['credentials.yaml']
```

Comment:
```suggestion

```

Reply:
Done in 85a36ccf7e801c47cce59c4f3fa7a90ec0ac563f